docs/src/main/asciidoc/includes/openapi/openapi-generator.adoc

```diff
@@ -20,7 +20,6 @@
 :keywords: helidon, {flavor-lc}, openapi, generator
 :feature-name: Helidon {flavor-uc} OpenAPI Generator
 // DO NOT CHANGE THE FOLLOWING - it's used as a minimum release that will not normally change with new releases of the OpenAPI generator
-:first-version-with-strong-helidon-support: 6.2.1
```

Comment:
I believe this change causes this warning:
```
[WARNING] include file not found: {sourcedir}/mp/openapi/OpenApiGeneratorSnippets.java
```

The remaining attributes are already defined in both downstream documents, if we remove the preamble the warning goes away.

Reply:
I have not looked at the source of the plug-in, but the warning seems incorrect.

My local runs of the `sitegen:serve`  have consistently shown that the snippet was properly included in the rendered page, the build warning notwithstanding. (BTW I have removed the now-obsolete comments just above the removed line and the warning still occurs.)

Reply:
The site generator parses the document headers as a separate invocation, it detects the end of headers and parses the corresponding fragment. 

The code that find the end of the document headers is wrong, it should look for the first empty line, but instead it looks for the fist line that starts with `==` ; it is also not a proper parser (E.g. does not skip comments etc.).